model/src/rollover.rs

```diff
@@ -5,6 +5,21 @@ use crate::Price;
 use crate::TxFeeRate;
 use crate::Usd;
 
+#[derive(Debug, Clone, Copy)]
+pub enum Version {
+    /// Version one of the rollover protocol
+    ///
+    /// This version cannot handle charging for "missed" rollovers yet, i.e. the hours to charge is
+    /// always set to 1 hour. This version is needed for clients that are <= daemon version
+    /// `0.4.7`.
```

Comment:
I understand why you're putting this comment here, but it's a bad sign that our model "knows" about daemon versions. Nothing to action, just a thought.

Reply:
I totally agree, I will see to given our versioning ADR a review later!

Reply:
From an utilitarian POV, I'd say that the model ha also 0.4.7 version :)